CONTRIBUTING.md

```diff
@@ -0,0 +1,165 @@
+# Contributing to Design System MCP Server
+
+Thank you for your interest in contributing to the Design System MCP Server! This guide will help you get started with contributing to this open-source project.
+
+## Code of Conduct
+
+By participating in this project, you agree to abide by our [Code of Conduct](CODE_OF_CONDUCT.md). Please read it before contributing.
+
+## How to Contribute
+
+### Reporting Issues
+
+- Use our [issue templates](.github/ISSUE_TEMPLATE/) to report bugs or request features
+- Search existing issues before creating a new one
+- Provide clear, detailed descriptions with examples when possible
+
+### Suggesting Enhancements
+
+- Open an issue using the feature request template
+- Explain the use case and expected behavior
+- Consider if the enhancement fits within the design system's scope
+
+### Contributing Documentation
```

Comment:
Almost everything from here to the end of this file is relevant to the docs repo, not this repo. So let's remove it. Overall make sure the file makes sense for this repo.